vitest.config.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://vitest.dev/config/
+ */
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+    test: {
+        deps: {
+            interopDefault: true
+        },
+        include: ['packages/**/test/**/*.test.ts']
```

Comment:
Can we set some basic coverageThreshold in vitest config? It automatically fails if coverage drops down.  
like in Jest e.g.: 
thresholds: {
        lines: 80,
        functions: 80,
        branches: 80,
        statements: 80
      }

Reply:
I would strongly advise against that because it won't help with the quality and will create a number of undesired side effects.

Reply:
This is common practice. Side effect only happens in code change not in test, this is just a test coverage measurement. Same measures tuned in sonar or Jenkins upon project.